src/main/java/com/fasterxml/jackson/databind/ser/std/StdJdkSerializers.java

```diff
@@ -48,12 +64,16 @@
         sers.put(Void.class, NullSerializer.instance);
         sers.put(Void.TYPE, NullSerializer.instance);
 
+        //And Sql Blob
+        sers.put(Blob.class, new SqlBlobSerializer());
```

Comment:
Move this down to next block, along with `java.sql.Timestamp` etc; that way it can be optional wrt Module system.